calico_node/startup/startup.go

```diff
@@ -817,8 +817,12 @@ func ensureDefaultConfig(ctx context.Context, cfg *apiconfig.CalicoAPIConfig, c
 			newFelixConf.Spec.LogSeverityScreen = defaultLogSeverity
 			_, err = c.FelixConfigurations().Create(ctx, newFelixConf, options.SetOptions{})
 			if err != nil {
-				log.WithError(err).WithField("FelixConfig", newFelixConf).Errorf("Error creating Felix global config")
-				return err
+				if conflict, ok := err.(cerrors.ErrorResourceUpdateConflict); ok {
```

Comment:
Didn't we decide that this needs to be an `ErrorResourceAlreadyExists`  because we're doing a Create?

Reply:
Yes we did, I must've mixed that one up, I'll double check the others too.